terminado/websocket.py

```diff
@@ -77,15 +104,25 @@ def open(self, url_component=None):
         self.terminal.clients.append(self)
 
         self.send_json_message(["setup", {}])
+        
+        self.__update_last_activity_time()
```

Comment:
Let's do this inside the `if self.keep_alive_time > 0` block; there's no need to do it when the pinging is disabled.